.circleci/config.yml

```diff
@@ -43,11 +43,11 @@ jobs:
             # Holiday dates specific to 2022 that will need to be updated next year
             # as per https://intranet.broadinstitute.org/keywords/holiday-calendar
             # and https://intranet.broadinstitute.org/hr/human-resources-policy-manual/time
-            HOLIDAYS=("12/23/2021" "01/17/2022" "02/21/2022" "04/18/2022" "05/30/2022" "07/04/2022"
-            "09/05/2022" "10/10/2022" "11/11/2022" "11/24/2022" "11/25/2022" "12/23/2022")
+            HOLIDAYS=("12/23/2021" "01/17/2022" "02/21/2022" "04/18/2022" "05/30/2022" "06/20/2022" "07/01/2022" "07/04/2022"
+            "09/02/2022" "09/05/2022" "10/10/2022" "11/11/2022" "11/24/2022" "11/25/2022" "12/23/2022")
 
             # List of Holidays the Broad is closed for every year
-            HOLIDAYSEACHYEAR=("01/01" "12/24" "12/25" "12/26" "12/27" "12/28" "12/29" "12/30" "12/31")
+            HOLIDAYSEACHYEAR=("01/01" "06/19" "12/24" "12/25" "12/26" "12/27" "12/28" "12/29" "12/30" "12/31")
```

Comment:
Note that the recurring dates above have been only for Broad closing at the end of a year. I'm not necessarily opposed to adding more dates to that list but we should probably be consistent if we do. For instance, July 4th, Labour Day, etc would be in the same bucket as June 19th.

Reply:
Added the other fixed dates to the recurring list in https://github.com/DataBiosphere/saturn-ui-prod-deploy/pull/29/commits/12c20204e0b5eda921322b27190f7c8169945b96.